x-pack/plugins/cases/public/components/create/owner_selector.tsx

```diff
@@ -104,12 +94,18 @@ const OwnerSelector = ({
     </EuiFormRow>
   );
 };
+
 OwnerSelector.displayName = 'OwnerSelector';
 
 const CaseOwnerSelector: React.FC<Props> = ({ availableOwners, isLoading }) => {
+  const defaultValue = availableOwners.includes(SECURITY_SOLUTION_OWNER)
+    ? SECURITY_SOLUTION_OWNER
+    : availableOwners[0] ?? SECURITY_SOLUTION_OWNER;
```

Comment:
this might be just for the types but why is the security solution the backup owner??

Reply:
Hmm, good point. Let's default it to Stack cases.